lab2/src/task1/Main.java

```diff
@@ -0,0 +1,99 @@
+package task1;
+
+import java.util.*;
+
+class Complex {
+
+    private int real;
+    private int imaginary;
+
+    // TODO: Add constructors
+
+    public Complex(int pReal, int pImg) {
+        real = pReal;
+        imaginary = pImg;
+    }
+
+    public Complex() {
+        this(0, 0);
+    }
+
+    public Complex(Complex obj) {
+        this.real = obj.real;
+        this.imaginary = obj.imaginary;
+    }
+
+    // TODO: Add getters and setters
+
+    public int getReal() {
+        return real;
+    }
+
+    public void setReal(int real) {
+        this.real = real;
+    }
+
+    public int getImaginary() {
+        return imaginary;
+    }
+
+    public void setImaginary(int imaginary) {
+        this.imaginary = imaginary;
+    }
+
+
+    // TODO: Add addWithComplex
+
+    void addWithComplex(Complex addMe) {
+        this.real += addMe.real;
+        this.imaginary += addMe.imaginary;
+    }
+
+    // TODO: Add showNumber
+
+    void showNumber() {
+        if (this.imaginary < 0) {
+            System.out.println(this.real + " - i * " + Math.abs(this.imaginary));
+        } else if (this.imaginary > 0) {
+            System.out.println(this.real + " + i * " + Math.abs(this.imaginary));
+        } else if (this.imaginary == 0) {
+            System.out.println(this.real);
+        }
+    }
+}
+
+public class Main {
```

Comment:
Don't put all the classes in a single file. When you push to git, I want to see how you structure your solution, and putting all the classes in a single file is not a good practice